core/templates/dev/head/dashboard/dashboard.html

```diff
@@ -41,6 +41,9 @@ <h1 class="stat-value"><[dashboardStats.average_ratings || 'N/A']></h1>
         <div class="total-plays stats-card">
           <p class="stat-description">Total Plays</p>
           <h1 class="stat-value"><[dashboardStats.total_plays]></h1>
+          <p ng-hide="(dashboardStats.total_plays - lastWeekStats.total_plays) == 0">
```

Comment:
Recommend moving this calculation to the controller, since you're repeating it here twice.